### PR TITLE
Add support for Kubernetes v1.13.0-alpha.1

### PR DESCRIPTION
**Pull request by mboersma**
**What this PR does / why we need it**:

See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md#v1130-alpha1

TODO:

- [x] upload Windows binary
- [x] ~Refactor cluster-proportional-autoscaler so its version can vary? (1.13.x wants version 1.3.0)~ acs-engine doesn't yet support this incubator component, so let's not block this PR on that.
- [x] Run 1.13 CircleCI job for verification (is there a chicken-and-egg issue here?)

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [x] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```release-note
Kubernetes: add support for v1.13.0-alpha.1
```

### CODE REVIEW

File: pkg/api/k8s_versions.go [hunk 1]
```diff
@@ -8,6 +8,38 @@ import (
...
 var k8sComponentVersions = map[string]map[string]string{
+	"1.13": {
+		"dockerEngine":                     "1.13.*",
+		"dashboard":                        "kubernetes-dashboard-amd64:v1.10.0",
+		"addon-resizer":                    "addon-resizer:1.8.1",
+		"heapster":                         "heapster-amd64:v1.5.3",
+		"metrics-server":                   "metrics-server-amd64:v0.2.1",
+		"coredns":                          "coredns:1.2.2",
+		"kube-dns":                         "k8s-dns-kube-dns-amd64:1.14.13",
+		"addon-manager":                    "kube-addon-manager-amd64:v8.8",
```
Comment (mboersma) [inline pkg/api/k8s_versions.go]: This is the only component bumped according to the release notes (besides cluster-proportional-autoscaler, which is still a TODO: for this PR).

**Comment by CecileRobertMichon** [comment]
Does upgrade to 1.13 work out-of-the-box?

**Comment by mboersma** [comment]
> Does upgrade to 1.13 work out-of-the-box?

Not yet. I'm investigating:

```
...
INFO[0006] Upgrading to Kubernetes version 1.13.0-alpha.1
 
FATA[0006] Error upgrading cluster: Upgrade to Kubernetes version 1.13.0-alpha.1 is not supported
```

**Pull request merged**